app/notifications/receive_notifications.py

```diff
@@ -28,6 +28,7 @@ def receive_mmg_sms():
         'DateRecieved': '2017-05-21+11%3A56%3A11'
     }
     """
+    current_app.logger.info("Inbound sms sender IP information {}".format(request.headers.getlist("X-Forwarded-For")))
```

Comment:
@yahoopete @servingUpAces are we okay putting this IP address in our logs? Given that it'll just be MMG/firetext, i'm not sure how much that classifies as PII